src/main/java/com/treasure/hunt/game/GameManager.java

```diff
@@ -5,62 +5,73 @@
 import com.treasure.hunt.strategy.tipster.Tipster;
 import com.treasure.hunt.strategy.seeker.Seeker;
 import com.treasure.hunt.view.in_game.View;
-import lombok.AllArgsConstructor;
 import org.locationtech.jts.geom.Coordinate;
 import org.locationtech.jts.geom.GeometryFactory;
 import org.locationtech.jts.geom.Point;
 
 import java.util.ArrayList;
 import java.util.List;
 
-@AllArgsConstructor
+/**
+ * This is the GameManager which should be started,
+ * to start a normal game.
+ */
 public class GameManager {
 
     // final variables
-    private final GeometryFactory gf = new GeometryFactory();
-    private final GameHistory gameHistory = new GameHistory();
-    private final Seeker seeker;
-    private final Tipster tipster;
-    private final List<Thread> threads;
+    protected final GeometryFactory gf = new GeometryFactory();
```

Comment:
Everything should be private unless required differently ? Whyy make it protected?